src/cascadia/TerminalSettingsModel/FontConfig.cpp

```diff
@@ -65,6 +68,42 @@ void FontConfig::LayerJson(const Json::Value& json)
         JsonUtils::GetValueForKey(fontInfoJson, FontFaceKey, _FontFace);
         JsonUtils::GetValueForKey(fontInfoJson, FontSizeKey, _FontSize);
         JsonUtils::GetValueForKey(fontInfoJson, FontWeightKey, _FontWeight);
+
+        if (fontInfoJson.isMember(JsonKey(FontFeaturesKey)))
+        {
+            std::unordered_map<hstring, uint32_t> featureMap;
+            const auto fontFeaturesJson = fontInfoJson[JsonKey(FontFeaturesKey)];
+            const auto featureNames = fontFeaturesJson.getMemberNames();
+            for (const auto& featureName : featureNames)

```

Comment:
@DHowett turns out that doing `for (const auto& [feature, val] : featureNames)` does not work but this method does, so I don't think we need a `ConversionTrait` anymore